### PR TITLE
chore: update request types supported

### PR DESCRIPTION
**Pull request by rachael-t**
### Description

Updates the request types supported by the webhook channel and fetch step to align with what is available/supported in the dashboard.

### Screenshots

https://docs-git-rt-webhook-channel-request-types-knocklabs.vercel.app/integrations/webhook/overview

Screenshot 2024-11-27 at 2 29 21 PM

https://docs-git-rt-webhook-channel-request-types-knocklabs.vercel.app/designing-workflows/fetch-function#building-a-request

Screenshot 2024-11-27 at 2 29 39 PM

### CODE REVIEW

File: content/integrations/webhook/overview.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ Learn more about how to use Knock webhook channels to send to custom destination
 
 ## Features and use cases
 
-You can use the Knock webhook channel type to build a custom channel that sends a webhook request to a configured endpoint. This endpoint can be static, or can be dynamically built using liquid variables during workflow run time. The Knock webhook channel supports PUT, POST, and GET requests, making it a flexible tool to use for a number of different use cases.
+You can use the Knock webhook channel type to build a custom channel that sends a webhook request to a configured endpoint. This endpoint can be static, or can be dynamically built using liquid variables during workflow run time. The Knock webhook channel supports GET, POST, PUT, DELETE, and PATCH requests, making it a flexible tool to use for a number of different use cases.
```
[the remark this answers is not in the document]
Comment (cellomatt) [reply content/integrations/webhook/overview.mdx]: image

[the remark this answers is not in the document]
Comment (cellomatt) [reply content/integrations/webhook/overview.mdx]: Wherever we land on this, I think we should also make updates to https://docs.knock.app/designing-workflows/fetch-function to make sure it includes all the available HTTP methods and that styling is consistent across these two pages for the `code` style (colored or not)

**Review by cellomatt — CHANGES_REQUESTED** [review]
We should also make updates to https://docs.knock.app/designing-workflows/fetch-function to make sure it includes all the available HTTP methods and that styling is consistent across these two pages for the `code` style (colored or not)

File: content/integrations/webhook/overview.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (rachael-t) [reply content/integrations/webhook/overview.mdx]: I love this actually. Thank you for thinking of this! 🙌

File: content/designing-workflows/fetch-function.mdx [hunk 2]
```diff
@@ -13,11 +13,11 @@ With the fetch function, you can acquire additional data for your channel step t
...
 
-- **Request method** - You can select one of `GET` (the default), `POST`, or `PUT`.
+- **Request method** - You can select one of <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-blue-100 dark:bg-transparent dark:border-blue-600">GET</span> (default) <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-green-100 dark:bg-transparent dark:border-green-600">POST</span> <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-yellow-100 dark:bg-transparent dark:border-yellow-500">PUT</span> <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-red-100 dark:bg-transparent dark:border-red-500">DELETE</span> or <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-purple-100 dark:bg-transparent dark:border-purple-500">PATCH</span>.
```
Comment (cellomatt) [inline content/designing-workflows/fetch-function.mdx:16]:
```suggestion
- **Request method** - You can select one of <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-blue-100 dark:bg-transparent dark:border-blue-600">GET</span> (default), <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-green-100 dark:bg-transparent dark:border-green-600">POST</span>, <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-yellow-100 dark:bg-transparent dark:border-yellow-500">PUT</span>, <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-red-100 dark:bg-transparent dark:border-red-500">DELETE</span>, or <span className="text-xs font-medium text-gray-600 dark:text-gray-300 border border-transparent font-mono rounded p-1 center bg-purple-100 dark:bg-transparent dark:border-purple-500">PATCH</span>.
```
nit, add commas after each request type

**Review by cellomatt — APPROVED** [review]
just one tiny nit. thanks for making this update!!

**Pull request merged**